test/test_other.py

```diff
@@ -7747,20 +7758,22 @@ def test_dlopen_rpath(self):
     os.mkdir('subdir')
 
     def _build(rpath_flag, expected, **kwds):
-      self.run_process([EMCC, '-o', 'subdir/libhello_dep.so', 'hello_dep.c', '-sSIDE_MODULE'])
+      self.run_process([EMCC, '-o', 'subdir/libhello_nested_dep.so', 'hello_nested_dep.c', '-sSIDE_MODULE'])
+      self.run_process([EMCC, '-o', 'subdir/libhello_dep.so', 'hello_dep.c', '-sSIDE_MODULE', 'subdir/libhello_nested_dep.so'] + rpath_flag)
```

Comment:
If you want to use the `.so` extensions in this test we should also rename `libhello.wasm` to `libhello.so` above.   Or use the `.wasm` extension here.  I don't really mind which.

Reply:
Makes sense. I renamed it to `.so` extension.